### DISCUSSION TITLE
View deployments from all projects in a single view

### DISCUSSION THREAD

**Discussion by joshiefishbein** [vercel/community#771] [Idea]
We have a view for seeing all deployments in a single project, can we merge all deployments for all projects into a single view? Or maybe all queued/actively building deployments from all projects in a single view?

I would sacrifice any/all filtering of repos and commits simply so I can see what my few containers are actively working on. It's not the greatest experience when I have to go hunting for the deployment that's in progress, say if a build is hanging up on something and blocking the rest of the deployment pipeline. 

It would also be a great place to show me the stat on how much total time my deployments stay queued and upsell me on more containers if you need a $$$ incentive.

When builds were failing in the CLE1 region on August 3, 2022, I had no idea what was going on (vercel-status had yet to be updated). My Vercel org has a dozen projects that are selectively deployed when sensing changes in 3 different repos. So I was going a little crazy for 2 hours with a dozen tabs open, clicking back and forth, canceling and retriggering deployments, before the status page had been updated that day. If I had a single pane for all queued deployments then, I would've probably noticed this was a Vercel issue and may have reached out to support sooner. This was an extreme situation, but I'm sure I would benefit from this view as an engineering manager on a daily basis.

**Comment by mikemclin** [comment]
I have a monorepo that has multiple apps that get deployed on push.  Hunting down which project is currently deploying is a frustrating experience.  A global deployments view would give me much better insight as to what is deploying, and which job would be next in the queue and so on.
